pootle_pytest/factories.py

```diff
@@ -41,3 +45,110 @@ class AgreementFactory(factory.django.DjangoModelFactory):
 
     class Meta(object):
         model = 'staticpages.Agreement'
+
+
+class DirectoryFactory(factory.django.DjangoModelFactory):
+
+    @factory.lazy_attribute
+    def pootle_path(self):
+        if self.parent is None:
+            return "/"
+        return (
+            "%s/%s"
+            % (self.parent.pootle_path.rstrip("/"),
```

Comment:
hmm - it might not be - but there was a situation that came up where it was - so i would rather err on the side of caution